recipes/default.rb

```diff
@@ -32,6 +32,16 @@ class Chef::Resource # rubocop:disable all
   # logrotate is a package dependency of rabbitmq-server
   package 'logrotate'
 
+  # dpkg, imma let you finish but don't start services automatically
+  # https://jpetazzo.github.io/2013/10/06/policy-rc-d-do-not-start-services-automatically/
```

Comment:
Correct, the hack will only apply to debian platform_families although other commits in that PR do fix the root issue on both RHEL and Debian platform_families